packages/Webkul/Core/src/Helpers/Laravel5Helper.php

```diff
@@ -306,44 +249,23 @@ private function haveDownloadableProduct(array $configs = [], array $productStat
         return $product->refresh();
     }
 
-    /**
-     * @param array $attributes
-     * @param array $states
-     *
-     * @return \Webkul\Product\Models\Product
-     */
     private function createProduct(array $attributes = [], array $states = []): Product
     {
         return factory(Product::class)->states($states)->create($attributes);
     }
 
-    /**
-     * @param int   $productId
-     * @param array $inventoryConfig
-     *
-     * @return void
-     */
     private function createInventory(int $productId, array $inventoryConfig = []): void
     {
         $I = $this;
-
         $I->have(ProductInventory::class, array_merge($inventoryConfig, [
             'product_id'          => $productId,
             'inventory_source_id' => 1,
-            'qty'                 => random_int(100, 666),
         ]));
```

Comment:
You accidentally removed the quantity ;)

Reply:
From my point of view quantity hasn't to be set here. It's set in the factory.
https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Product/src/Database/Factories/ProductInventoryFactory.php
And this is what @thyseus has done in the factory.

So all is fine with that.

Reply:
You are right, it was my fault. I created a Simple product using haveProduct() and had have no inventory. It was because I created a new channel before, and did not add the InventorySource to it. I will think about adapting the ChannelFactory accordingly, perhaps nicer than that one, though: 

`DB::table('channel_inventory_sources')->insert(['channel_id' => $channel->id, 'inventory_source_id' => 1]);`